.github/dockerfiles/ov_build/ubuntu_22_04_riscv_xuantie/Dockerfile

```diff
@@ -0,0 +1,102 @@
+        python3-venv \
+        python3-distutils \
+        # Compilers
+        gcc \
+        g++ \
+        # xuantie-gnu-toolchain build dependencies
+        autoconf \
+        automake \
+        autotools-dev \
+        libmpc-dev \
+        libmpfr-dev\ 
+        libgmp-dev \
+        gawk \
+        build-essential \
+        bison \
+        flex \
+        texinfo \
+        gperf \
+        libtool \
+        patchutils \
+        bc \
+        zlib1g-dev \
+        libexpat-dev \
+        && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install RISC-V native debian packages
+RUN echo deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy main restricted > riscv64-sources.list && \
+    echo deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy-updates main restricted >> riscv64-sources.list && \
+    echo deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy universe >> riscv64-sources.list && \
+    echo deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy-updates universe >> riscv64-sources.list && \
+    echo deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy multiverse >> riscv64-sources.list && \
+    echo deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy-updates multiverse >> riscv64-sources.list && \
+    echo deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ jammy-backports main restricted universe multiverse >> riscv64-sources.list && \
+    echo deb [arch=amd64] http://security.ubuntu.com/ubuntu/ jammy-security main restricted >> riscv64-sources.list && \
+    echo deb [arch=amd64] http://security.ubuntu.com/ubuntu/ jammy-security universe >> riscv64-sources.list && \
+    echo deb [arch=amd64] http://security.ubuntu.com/ubuntu/ jammy-security multiverse >> riscv64-sources.list && \
+    echo deb [arch=riscv64] http://ports.ubuntu.com/ubuntu-ports/ jammy main >> riscv64-sources.list && \
+    echo deb [arch=riscv64] http://ports.ubuntu.com/ubuntu-ports/ jammy universe >> riscv64-sources.list && \
+    echo deb [arch=riscv64] http://ports.ubuntu.com/ubuntu-ports/ jammy-updates main >> riscv64-sources.list && \
+    echo deb [arch=riscv64] http://ports.ubuntu.com/ubuntu-ports/ jammy-security main >> riscv64-sources.list && \
+    mv riscv64-sources.list /etc/apt/sources.list.d/
+
+RUN dpkg --add-architecture riscv64 && \
+    apt-get update -o Dir::Etc::sourcelist=/etc/apt/sources.list.d/riscv64-sources.list && \
+    apt-get install -y --no-install-recommends libpython3-dev:riscv64
+
+# build xuintie toolchain
+ARG XUANTIE_VERSION="V2.8.1"
+ARG XUANTIE_REPO="https://github.com/XUANTIE-RV/xuantie-gnu-toolchain"
+ARG XUINTIE_PATH="/opt/riscv"
+ARG XUINTIE_TMP_PATH="/tmp/xuantie"
+ARG XUINTIE_SRC="/tmp/xuantie/src"
+    
+RUN mkdir -p ${XUINTIE_TMP_PATH} && cd ${XUINTIE_TMP_PATH} && \
+    git clone --branch ${XUANTIE_VERSION} --depth 1 ${XUANTIE_REPO} ${XUINTIE_SRC} && cd ${XUINTIE_SRC} && \
+    ./configure --prefix=${XUINTIE_PATH} --disable-gdb && \
+    make linux -j$(nproc) && make install && \
```

Comment:
If we want to use `qemu-riscv64`, should we add `qemu` to the command `make` here?
As far as I remember, we need to add explicitly `qemu` to enable this build. Correct me please if I'm wrong